src/core/ImportRecord.ts

```diff
@@ -17,8 +18,8 @@ export default class ImportRecord {
         }
     }
 
-    // Is this import an external dependency (i.e. is it under node_modules)?
+    // Is this import an external dependency (i.e. is it under node_modules or outside the rootPath)?
     get isExternal() {
-        return this.filePath.split(path.sep).indexOf('node_modules') != -1;
+        return this.filePath.split(path.sep).indexOf('node_modules') != -1 || !this.filePath.startsWith(getOptions().rootDir);
```

Comment:
Can you also check whether the `rawImport` starts with `'.'`?  If the import is a relative path then I think by definition it should be treated as internal.  (I think this is consistent with usage in a Rush monorepo, right?  If you want to refer to another package it should not be a relative path, but rather should start withe the name of the package.)

Reply:
Added the extra check, and made the logic a bit more verbose to better document what is going on.